scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -69,7 +69,13 @@ import scala.compat.Platform.EOL
     allowXmlLiterals: Boolean,
     // What kind of separator is necessary to split top-level statements?
     // Normally none is required, but scripts may have their own rules.
-    toplevelSeparator: String
+    toplevelSeparator: String,
```

Comment:
These are binary breaking changes that would require us to release Scalameta v5.0.0. The trick to add new fields while keeping binary compatibility is to add private vars like here https://github.com/scalameta/scalameta/blob/049c9a823404c3b303597f157eac50bd67c796e1/scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala#L78-L92

This requires a fair amount of boilerplate but that's life

Reply:
okay :+1:

Reply:
@olafurpg I rebased on top of master, could you double check I added new parameters correctly?